Package.swift

```diff
@@ -23,6 +23,7 @@ let package = Package(
         ),
         // The tests depend on valid protos via gradle
         // ./gradlew -p wire-library :wire-runtime-swift:generateTestProtos
+        // ./gradlew -p wire-library :wire-runtime-swift:generateSampleProtos
```

Comment:
The comment makes it sound like the tests would not pass without having run `generateSampleProtos` but is that really the case? If not, no need for the comment I think ?